### PR TITLE
Stub Warden server

### PR DESCRIPTION
**Pull request by purujit**
Just creating the stub in this PR. Implementation will come as a follow-up.

### CODE REVIEW

File: proto/target/epoch_broadcaster/epoch_broadcaster.rs
Comment (purujit) [inline proto/target/epoch_broadcaster/epoch_broadcaster.rs:1]: Why are the generated files checked in?

Comment (tamer-eldeeb) [reply proto/target/epoch_broadcaster/epoch_broadcaster.rs:1]: No good reason :) https://github.com/chardonnaydb/chardonnay/pull/58

**Comment by purujit** [comment]
@tamer-eldeeb ptal when you get a chance.

File: proto/src/warden.proto [hunk 1]
```diff
@@ -19,12 +19,15 @@ service Warden {
...
-   repeated RangeId range = 1;
+   int64 version = 1;
+   repeated RangeId range = 2;
 }
 
 message IncrementalAssignment {
-    repeated RangeId load = 1;
-    repeated RangeId unload = 2;
+    int64 version = 1;
+    int64 previous_version = 2;
```
Comment (tamer-eldeeb) [inline proto/src/warden.proto:28]: Could you please add some documentation as to what these versions mean? (either here or somewhere in the warden package). I have some guesses as to how they're supposed to work but I'm not sure :)

Comment (purujit) [reply proto/src/warden.proto:36]: Done.

File: warden/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+use server::run_warden_server;
+// use server::run_warden_server;
```
Comment (tamer-eldeeb) [inline warden/src/main.rs:2]: nit: dead code

**Review by tamer-eldeeb — APPROVED** [review]

File: warden/src/main.rs (hunk 2 above)
Comment (purujit) [reply warden/src/main.rs]: Done.

**Pull request merged**